sdk/src/main/scala/io/horizen/account/state/McAddrOwnershipMsgProcessor.scala

```diff
@@ -31,7 +31,7 @@ import scala.jdk.CollectionConverters.seqAsJavaListConverter
 
 trait McAddrOwnershipsProvider {
```

Comment:
for future readability let's add ontop of this class a comment to describe the datasctructure used in the storage
(the two linkedlist etc...)

Reply:
Added